caffeine@patapon.info/extension.js

```diff
@@ -98,7 +98,6 @@ const DBusSessionManagerInhibitorIface = '<node>\
 const DBusSessionManagerInhibitorProxy = Gio.DBusProxy.makeProxyWrapper(DBusSessionManagerInhibitorIface);
 
 const IndicatorName = 'Caffeine';
-const TimerMenuName = _('Caffeine timer');
```

Comment:
I'm not sure I understand the motivation for this? This removes the one instance of the translation, instead translating each time the label is created / updated.

Reply:
For some reasons, the translation did not work on my previous test when I was using constants `IndicatorLabel` and `TimerMenuName`, so I simply removed them. I will update the code and redo some test.